tests/daemon/test-persistent-cache.c

```diff
@@ -1081,7 +1060,7 @@ test_persistent_cache_get_offset_wont_update_timestamps_if_it_isnt_supposed_to (
 
```

Comment:
Alternatively, you could remove this test case entirely at your discretion since it's effectively testing the functionality of a parameter that has been removed.